MapleServer2/Database/Classes/DatabaseItem.cs

```diff
@@ -59,15 +59,28 @@ public long Insert(Item item)
             transfer_flag = item.TransferFlag,
             blackmarket_category = item.BlackMarketCategory,
             transparency_badge_bools = JsonConvert.SerializeObject(item.TransparencyBadgeBools),
-            unlock_time = item.UnlockTime
+            unlock_time = item.UnlockTime,
+            ugc_uid = item.UGC == null ? null : (int?) item.UGC.Uid,
```

Comment:
It's for casting that ugc_uid can be null since the true statement is null.

It's either `item.UGC == null ? null : (int?) item.UGC.Uid,` or `item.UGC == null ? (int?) null : (int) item.UGC.Uid,`

Reply:
Compiler says: **There is no implicit conversion between 'null' and 'int'**